doc/tutorials/langevin_dynamics/langevin_dynamics.ipynb

```diff
@@ -4,78 +4,61 @@
    "cell_type": "markdown",
```

Comment:
<p>not the signature like in the solution. Also probably we need to describe what these parameters mean. e.g. pid: list of integers corresponding to particle ids</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>